packages/web-components/@sfx/sayt/src/sayt.ts

```diff
@@ -42,6 +43,19 @@ export default class Sayt extends LitElement {
    * The minimum length of the search term required before a SAYT request will be made with it.
    */
   @property({ type: Number, reflect: true }) minSearchLength = 3;
+  /**
+   * The debounce delay in millilseconds.
+   */
+  @property({ type: Number, reflect: true }) debounce = 300;
+  /**
+   * The debouncedRequestSaytProducts method that will wrap the requestSaytProducts method in a debounce.
+   */
+  debouncedRequestSaytProducts: requestSaytType;
```

Comment:
These declarations won't be necessary with the new approach, but you'll have to cast the functions as `ReturnType<typeof debounce>` in the tests to be able to use `flush()`.